node_cli/core/node.py

```diff
@@ -116,8 +115,13 @@ def restore(backup_path, env_filepath):
     if env_params is None:
```

Comment:
I think it's better to use get_node_env here.

Reply:
replaced